.vib/common/goss/templates/check-linked-libraries.yaml

```diff
@@ -12,5 +12,5 @@ command:
         {{ $exclude_paths = (.Vars.linked_libraries.exclude_paths | join "|") }}
     {{ end }}
     exec: export BITNAMI_ROOT_DIR={{ .Vars.root_dir }} && export EXCLUDE_PATHS='{{ $exclude_paths }}' && ./common/goss/scripts/check-linked-libraries.sh
-    timeout: 20000
```

Comment:
@FraPazGal @joancafom Should we increase the timeout for all assets? Or should we instead add support for a customizable timeout?

In [this PR](https://github.com/bitnami/containers/pull/30858/files#diff-482f3343f255b156bae488ed2c775287ae0d7e03bc6b637c046234ea561f4212) I implemented the option to include `linked_libraries.timeout` in the vars.yaml file for those cases were a bigger timeout is required.

What do you think?